app/graphql/loaders/media_characters_loader.rb

```diff
@@ -0,0 +1,6 @@
+class Loaders::MediaCharactersLoader < Loaders::FancyLoader
+  from MediaCharacter
+
+  sort :created_at
+  sort :updated_at
```

Comment:
can you add a sort for role? We want to display main characters at the top

Reply:
hah, was thinking of adding that just wasn't sure.